apel/db/apeldb.py

```diff
@@ -60,7 +60,14 @@ def test_connection(self):
         '''Connects to the database then closes the connection.'''
         pass
 
-    def load_records(self, record_list, source):
+    def check_versions(self):
+        '''
+        Verify that the database schema and codebase versions match requirements
+        given in apel/__init__.py
+        '''
```

Comment:
```suggestion
        '''Verify that the database schema and codebase versions are equal'''
```

The word "requirements" makes me think of `requirements.txt`